com.unity.cinemachine/Tests/Runtime/HDRP/CheckPhysicalCameraProperties.cs

```diff
@@ -0,0 +1,106 @@
+#if TEST_CINEMACHINE_HDRP
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Unity.Cinemachine.Tests.HDRP
```

Comment:
We don't want a separate HDRP folder.  Just put it in Runtime, and adjust the namespace to match the other things in Runtime

Reply:
Fixed